test/integration/ovs/ofctrl_test.go

```diff
@@ -553,12 +553,12 @@ func TestBundleWithGroupAndFlow(t *testing.T) {
 	field2 := binding.NewRegField(2, 0, 31)
 	field3 := binding.NewRegField(3, 0, 31)
 	group := bridge.CreateGroup(groupID).
-		Bucket().Weight(100).
+		Bucket().Weight(openflow.DefaultWeight).
```

Comment:
I feel no need to make openflow.DefaultWeight public just for usage here. But no strong opinion.

Reply:
Done